options/locale/locale_en-US.ini

```diff
@@ -923,12 +923,16 @@ repo_and_org_access = Repository and Organization Access
 permissions_public_only = Public only
 permissions_access_all = All (public, private, and limited)
 permission_not_set = Not set
+permission_not_set_helper = No public access to this unit
 permission_no_access = No Access
 permission_read = Read
 permission_write = Read and Write
 permission_anonymous_read = Anonymous Read
+permission_anonymous_read_helper = Non-signed-in users can read
 permission_everyone_read = Everyone Read
+permission_everyone_read_helper = Signed-in users can read
```

Comment:
It should also mention that `(for issues and PRs, read also means being able to create new)`